climpred/bootstrap.py

```diff
@@ -1,10 +1,58 @@
 import numpy as np
 import xarray as xr
 
-from .prediction import compute_perfect_model, compute_persistence_pm
+from .prediction import (compute_perfect_model, compute_persistence,
+                         compute_reference)
 from .stats import DPP, xr_varweighted_mean_period
 
 
+def resample_uninit_dple_from_lens(dple, lens):
```

Comment:
this is a bit quick and dirty. I wanted to have a LENS with lead dimension, so I bootstrapped. I think its nice to see the skill increase for ininitialized over lead years when not detrended. 
Didn't find the line in Boer et al. 2016 DCPP paper where they say for uninit lead=0 or 1 is enough.

Reply:
this is an alternative approach to `compute_uninitialized`. ends up being the same if we restrict ourselves then to .isel(lead=0)

Reply:
to reconciliate the two bootstrap functions we could also just have one argument which would the function of how to create an uninitialized ds/dple.